docs/introduction/Base-App.rst

```diff
@@ -92,6 +96,14 @@ This interface defines two methods:
 - ``Serializer serializer()`` - returns the class responsible to parse and write the object through Scorex Reader and Writer, which are wrappers on byte streams
 
 The SDK provides basic serializer interfaces for its objects (for example `BoxDataSerializer <https://github.com/HorizenOfficial/Sidechains-SDK/blob/master/sdk/src/main/java/com/horizen/box/BoxSerializer.java>`_ for BoxData, `TransactionSerializer <https://github.com/HorizenOfficial/Sidechains-SDK/blob/master/sdk/src/main/java/com/horizen/transaction/TransactionSerializer.java>`_ for Transactions), ready to be extended when writing specific custom serializers.
+All other serializers must implement the ScorexSerializer interface.
+
+This interface  two methods:
```

Comment:
This interface two methods: 

^ The above doesn't make sense. Do you mean This interface utilizes two methods or something similar?